source/includes/global-groovy-variables.rst

```diff
@@ -82,7 +78,7 @@
 .. |TenantService| replace:: :javadoc_base_url:`TenantService <profile/org/craftercms/profile/api/services/TenantService.html>`
 .. |AuthenticationService| replace:: :javadoc_base_url:`AuthenticationService <profile/org/craftercms/profile/api/services/AuthenticationService.html>`
 .. |AuthenticationManager| replace:: :javadoc_base_url:`AuthenticationManager <profile/org/craftercms/security/authentication/AuthenticationManager.html>`
-.. _TextEncryptor: http://docs.spring.io/autorepo/docs/spring-security/4.0.3.RELEASE/apidocs/org/springframework/security/crypto/encrypt/TextEncryptor.html
+.. |TextEncryptor| replace:: See ``TextEncryptor`` under ``org.springframework.security.crypto.encrypt`` in the Spring Security apidocs
 .. _Logger: http://www.slf4j.org/api/org/slf4j/Logger.html
-.. _XMLConfiguration: https://commons.apache.org/proper/commons-configuration/javadocs/v1.10/apidocs/org/apache/commons/configuration/XMLConfiguration.html
+.. |XMLConfiguration| replace:: See ``XMLConfiguration`` under ``org.apache.commons.configuration2`` in the Apache Commons apidocs
```

Comment:
Should we do the same here? https://commons.apache.org/proper/commons-configuration/index.html